### ISSUE TITLE
PreLoad conditions are not called

### ISSUE THREAD

**Issue by AlexandreRoba** [adrai/node-cqrs-domain#82]
Hello,

I have a domain with one aggregate and two commands.
I have defined two preload conditions for each commands.
I can see on the getInfo for each commands both preload conditions.
![image](https://cloud.githubusercontent.com/assets/60746/18054363/d09d033a-6e03-11e6-96bd-58f707aa9b0f.png)
When I submit the command matching the preload condition commands, none of the preload function are called. I have added logging and breakpoints. None of them are called or stopped at.
the preload condition are qui simple. I'm for exemple checking for a meta with a userId:
```javascript
module.exports = require('cqrs-domain').definePreLoadCondition({
    name: 'createNoise',
    description:'Create noise require authentication'
}, function (data) {
    let meta = data.meta || {};
    let userId = meta.userId || null;
    if(!userId){
        return new Error('Authentication is required');
    }
});
```
Any idea?

thanks for your help,

Alex

**Comment by AlexandreRoba** [comment]
Hello,

After a little more investigation I find out that setting the priority did resolve the issue. Isn't this suppose to be optional? Nevertheless it now works with the priority set.

thanks,

Alex.

**Comment by adrai** [comment]
Yes the priority is optional (defaults to Infinity), it should work without priority too.

**Comment by repkins** [comment]
Hello, @adrai!

I just also encountered with this issue by its nature, and, as there OP pointed cause of this by setting priority, for me does not solve this solution.

I have made own investigation by debugging to figure out how it accually handles pre-load conditions of command. and I have cause of this. The problem is place where it checks for defined pre-load conditions by checking it's length of pre-load conditions array in particular command instance. Acually it looks in `preConditions` array instead of `preLoadConditions` which means it apply pre-load conditions only if pre conditions is present in command.
![screenshot_2017-01-20_11-32-25](https://cloud.githubusercontent.com/assets/7951569/22143842/3744bc38-df04-11e6-9ad3-c9af463f1b79.png)
